OneSignalXCFramework.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+    s.name             = "OneSignalXCFramework"
+    s.version          = "2.14.3"
+    s.summary          = "OneSignal push notification library for mobile apps."
+    s.homepage         = "https://onesignal.com"
+    s.license          = { :type => 'MIT', :file => 'LICENSE' }
+    s.author           = { "Joseph Kalash" => "joseph@onesignal.com", "Josh Kasten" => "josh@onesignal.com" , "Brad Hesse" => "brad@onesignal.com"}
+    
+    s.source           = { :git => "https://github.com/OneSignal/OneSignal-iOS-SDK.git", :tag => s.version.to_s }
+    s.platform     = :ios
+    s.requires_arc = true
+    
+    s.ios.vendored_frameworks = 'iOS_SDK/OneSignalSDK/OneSignal_XCFramework/OneSignal.xcframework'
```

Comment:
It seems `vendored_frameworks` supports an array:
* https://www.rubydoc.info/github/CocoaPods/Core/Pod%2FSpecification%2FDSL:vendored_frameworks=
* https://guides.cocoapods.org/syntax/podspec.html#group_multi_platform_support

If this works it will allow us to only have to keep one cocoapod updated.

Reply:
Tested this and it results in both being in the pod project.
This somehow didn't create a duplicate error when building or running.  It seems cocoapods might be doing something to dedup by just picking one as it does result if both are imported manually outside of cocoapods, but we shouldn't be relying on it.
![image](https://user-images.githubusercontent.com/645861/88865440-c1944d00-d1bc-11ea-86a6-6a474708162d.png)

We will switch to only use xcframework in the future anyway. Once cocoapods 1.10 is released with it's [Revamped XCFramework Integration](http://blog.cocoapods.org/CocoaPods-1.10.0-beta/)